paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUIScreenshotTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.stripe.android.paymentsheet.verticalmode
+
+import com.stripe.android.lpmfoundations.luxe.SupportedPaymentMethod
+import com.stripe.android.model.PaymentMethodFixtures
+import com.stripe.android.paymentsheet.DisplayableSavedPaymentMethod
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.screenshottesting.SystemAppearance
+import com.stripe.android.utils.MockPaymentMethodsFactory
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.kotlin.mock
+
+internal class PaymentMethodVerticalLayoutUIScreenshotTest {
+    @get:Rule
+    val paparazziRule = PaparazziRule(
```

Comment:
```suggestion
    val paparazziRule = PaparazziRule()
```

The extra params are not needed.

Reply:
Guessing this was just copy/pasted from somewhere else. I removed all of them here: https://github.com/stripe/stripe-android/pull/8617/files